tools/utility/dockerise/run.py

```diff
@@ -97,6 +97,7 @@ def _run(**kwargs):
             "--interactive",
             "--env",
             "EDITOR={}".format(os.environ.get("EDITOR", "nano")),
+            "--privileged",
```

Comment:
This seems wrong and bad. @Y0sh1dk thoughts?

Reply:
This allows the image compressor to access the GPU hardware it needs to do compressions uber fast ... now that I think about it, this probably isnt needed in this PR

Reply:
The only time I have had to use `--privileged` is for docker-in-docker. If this is needed for the container to access the GPU hardware then we should probably modify `./b run` and `@run_on_docker` to allow you to pass a `--priviledged` flag or similar so we could do `./b run --privileged visualmesh`. But it should definitely not be on by default, privileged container = big bad

Reply:
It lets you access stuff you wouldn't normally be able to access. It's true it'll give you access to fancy stuff like the intel integrated GPU to do compression